kolena/_utils/validators.py

```diff
@@ -20,3 +20,8 @@ class ValidatorConfig:
     arbitrary_types_allowed = True
     smart_union = True
     extra = Extra.allow  # do not fail when unrecognized values are provided
+
+
+def validate_not_blank(field: str):
+    if not bool(field and not field.isspace()):
+        raise ValueError("Names must be non empty")
```

Comment:
The function name doesn't indicate that this is specific to `name` fields, but the exception assumes this is a `name` field. Which is correct?

Reply:
Updated the error to not mentioned name as this could be used for other fields later on.

Reply:
If a user sees the new "Field must be non-empty" exception, how do they know what to fix?

Reply:
Updating this to have a clearer error messages

Reply:
Updated to the approach we discussed offline.